lib/cdo/test_run_utils.rb

```diff
@@ -26,10 +26,110 @@ def self.run_blockly_core_tests
     end
   end
 
-  def self.run_dashboard_tests(parallel: false)
+  # Setup method to prepare test databases for parallel dashboard tests,
+  # which can be called from the QA pass (the test machine) or CI (Drone).
+  def self.setup_dashboard_tests_parallel(upload_seed_data: false)
+    # Parallel tests don't seem to run more quickly over 16 processes.
+    ENV['PARALLEL_TEST_PROCESSORS'] = '16' if RakeUtils.nproc > 16
```

Comment:
This would only happen if we've configured `PARALLEL_TEST_PROCESSORS` too high in the "prepare_ci_tests.sh" script, right? Should we warn or fail if we've configured it too high, rather than silently capping it?